src/Components/Restaurants.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import List from '@material-ui/core/List/index'
+import ListItem from '@material-ui/core/ListItem/index';
+import Restaurant from './Restaurant.js';
+
+
+class Restaurants extends Component {
```

Comment:
If you only have `render()`, use [stateless component](https://hackernoon.com/react-stateless-functional-components-nine-wins-you-might-have-overlooked-997b0d933dbc)

Reply:
removed